perses/plugindefinition.yaml

```
@@ -0,0 +1,20 @@
+apiVersion: greenhouse.sap/v1alpha1
+kind: PluginDefinition
+metadata:
+    name: perses
+spec:
+    version: 0.1.0
+    displayName: Perses
+    description: "Perses is a dashboard tooling to visualize metrics and traces produced by observability tools such as Prometheus/Thanos/Jaeger"
+    docMarkDownUrl: https://raw.githubusercontent.com/cloudoperators/greenhouse-extensions/main/perses/README.md
+    icon: https://raw.githubusercontent.com/cloudoperators/greenhouse-extensions/main/perses/logo.png
+    helmChart:
+        name: perses
+        repository: oci://ghcr.io/cloudoperators/greenhouse-extensions/charts
+        version: '0.7.4'
+    options:
+        - default: true
+          description: "Deploys a sidecar container to watch all configmaps in the cluster and filters out the ones with a label `perses.dev/resource: true`. The files defined in those configmaps are written to a folder and this folder is accessed by Perses. Changes to the configmaps are continuously monitored and reflected in Perses within 10 seconds"
+          name: perses.sidecar.enabled
+          required: true
+          type: bool
```

The indentation of yaml looks a bit off. 
```suggestion
kind: PluginDefinition
metadata:
  name: perses
spec:
  version: 0.1.0
  displayName: Perses
  description: "Perses is a dashboard tooling to visualize metrics and traces produced by observability tools such as Prometheus/Thanos/Jaeger"
  docMarkDownUrl: https://raw.githubusercontent.com/cloudoperators/greenhouse-extensions/main/perses/README.md
  icon: https://raw.githubusercontent.com/cloudoperators/greenhouse-extensions/main/perses/logo.png
  helmChart:
      name: perses
      repository: oci://ghcr.io/cloudoperators/greenhouse-extensions/charts
      version: '0.7.4'
  options:
    - default: true
      description: "Deploys a sidecar container to watch all configmaps in the cluster and filters out the ones with a label `perses.dev/resource: true`. The files defined in those configmaps are written to a folder and this folder is accessed by Perses. Changes to the configmaps are continuously monitored and reflected in Perses within 10 seconds"
      name: perses.sidecar.enabled
      required: true
      type: bool
```